src/omnicore/doc/release-notes.md

```diff
@@ -1 +1,236 @@
+Compatibility
+-----------
+
+Omni Core can be used as replacement for Bitcoin Core 0.10.0, 0.10.1, 0.10.2 and 0.10.3, and switching between Omni Core and Bitcoin Core is fully supported.
+
+Imported changes and notes
+==========================
+
+Upgrade to Bitcoin Core 0.10.3
+------------------------------
+
+The underlying base of Omni Core was upgraded to Bitcoin Core 0.10.3.
+
+Please see the following release notes for further details:
+
+- https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.10.0.md
+- https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.10.1.md
+- https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.10.2.md
+- https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.10.3.md
+
+Headers-first synchronization
+-----------------------------
+
+With the upgrade to Bitcoin Core 0.10 headers-first synchronization is supported and blocks are downloaded in parallel.
+
+Blocks are no longer stored in order on disk, and as result, the block files and databases are not backwards-compatible with older versions of Bitcoin Core prior 0.10.
+
+Dust threshold values
+---------------------
+
+The default `"minrelaytxfee"` was raised from `0.00001 BTC` to `0.00005 BTC`in Bitcoin Core 0.10.3 as temporary measure against massive memory pool bloat.
+
+This has an impact on the output values of Omni transactions, which are chosen to be as low as possible. As per default, Omni transactions created with Master Core had output values between `0.00000546 BTC` and `0.00000882 BTC`, whereby the new output values are between `0.0000273 BTC` and `0.0000441 BTC`.
+
+To continue to create transactions with lower values, start Omni Core with `-minrelaytxfee=0.00001` or add the following to your `bitcoin.conf`:
+```
+minrelaytxfee=0.00001
+```
+
+A lower `"minrelaytxfee"` may result in higher memory consumption, but is usually safe to use.
+
+Rebranding to Omni Core
+-----------------------
+
+Master Core was re-branded to Omni Core on all levels:
+
+- `mastercored`, `mastercore-cli` and `mastercore-qt` were renamed to `omnicored`, `omnicore-cli` and `omnicore-qt`
+- the debug log file `mastercore.log` was renamed  to `omnicore.log`
+- the hardcoded token SP#1 was renamed from `"MasterCoin"` to `"Omni"`
+- the hardcoded token SP#2 was renamed from `"Test MasterCoin"` to `"Test Omni"`
+- the user interface now refers to `"Omni"`, `"OMNI"`, `"Test Omni"` or `"TOMNI"` instead of `"Mastercoin"`, `"MSC"`, `"Test Mastercoin"` or `"TMSC"`
+
+Incompatible API changes
+------------------------
+
+The field `"subaction"` in the RPC response for traditional DEx orders of `"gettransaction_MP"` as well as `"listtransaction_MP"` was renamed to `"action"`.
+
+The values associated with `"action"` were renamed from `"New"`, `"Update"`, `"Cancel"` to `"new"`, `"update"`, `"cancel"`.
+
+Notable changes
+===============
+
+Feature activations
+-------------------
+
```

Comment:
How about this:

**Feature activations**

Omni Core 0.0.10 introduces feature activations, a new concept that allows the Omni team to decouple the release process from the process of making new features live.

Prior to 0.0.10, the block height that a new feature would be made live was hard coded, placing substantial limits on both the frequency of releases and our flexibility in introducing new features.  The feature activations system allows the release process and activation of features to be separated by allowing the Omni team to activate features with a special Omni transaction instead of with a hard coded block height within a specific release.

The activations system imposes set limits on when and by whom new features can be activated.  A minimum 2,048 block notice period (roughly 2 weeks) is enforced to give users with older incompatible clients time to upgrade, and the activation message must be signed by at least 4 of the 5 nominated members of the team in order to be accepted.  These members are as follows:

* dexx - Project maintainer, developer
* Zathras - Project maintainer, developer
* Adam Chamely - Project maintainer, developer
* J.R. Willett - Founder and Board Member
* Craig Sellars - Technologist and Board Member

Should a particular client version not support a new feature when it is activated, the client will shut down to prevent providing inaccurate data (whilst this behaviour can be overridden, it is strongly recommended against).  In this case the client should be upgraded.

The UI will display warnings for pending activations and daemon users can view pending and completed activations via the ```omni_getactivations``` RPC call.